### PR TITLE
Added historical section for Nagivation Timing 1 object

### PR DESCRIPTION
**Pull request by plehegar**
Proposal for resolving #22

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -965,6 +965,381 @@
...
+<a href="http://www.w3.org/TR/html5/infrastructure.html#fetching-resources">fetching</a> the resource.</p>
+
+</dd>
+<dt>readonly attribute unsigned long long domainLookupStart</dt>
+<dd>
+
+<p>This attribute must return the time immediately before the user agent starts the domain name
+lookup for the current document.
+If a <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec8.html#sec8.1">persistent
+connection</a> [[!RFC 2616]] is used or the current document
```
Comment (igrigorik) [inline index.html]: s/[[!RFC 2616]]/[[!RFC2616]] (no space)

File: index.html [hunk 2]
```diff
@@ -965,6 +965,381 @@
...
+this attribute must return the same value as <a href="#widl-PerformanceTiming-fetchStart">fetchStart</a>. </p>
+
+</dd>
+<dt>readonly attribute unsigned long long domainLookupEnd</dt>
+<dd>
+
+<p>This attribute must return the time immediately after the user agent
+finishes the domain name lookup for the current document.
+If a <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec8.html#sec8.1">persistent
+connection</a> [[!RFC 2616]] is used or the current document
```
Comment (igrigorik) [inline index.html]: same as above

File: index.html [hunk 3]
```diff
@@ -965,6 +965,381 @@
...
+</div>
+
+</dd>
+<dt>readonly attribute unsigned long long connectStart</dt>
+<dd>
+
+<p>This attribute must return the time immediately before the user agent start
+establishing the connection to the server to retrieve the document.
+If a <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec8.html#sec8.1">persistent
+connection</a> [[!]RFC2616]] is used or the current document is retrieved from <a
```
Comment (igrigorik) [inline index.html]: [[!] < typo?

File: index.html [hunk 4]
```diff
@@ -965,6 +965,381 @@
...
+
+</dd>
+<dt>readonly attribute unsigned long long connectEnd</dt>
+<dd>
+
+<p>This attribute must return the time immediately after the user agent
+finishes establishing the connection to the server to retrieve the current document.
+If a <a
+href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec8.html#sec8.1">persistent
+connection</a> [[!RFC 2616]] is used or the current document is retrieved from <a
```
Comment (igrigorik) [inline index.html]: extra space in rfc

**Comment by igrigorik** [comment]
A few linking nitpicks, but otherwise lgtm!

**Comment by toddreifsteck** [comment]
I think we need to add serializer to both of the original PerformanceTiming and PerformanceNavigation interfaces. They both implement it in Firefox and Microsoft Edge.

**Comment by toddreifsteck** [comment]
Otherwise, LGTM

**Comment by plehegar** [comment]
@toddreifsteck Good catch!

**Comment by plehegar** [comment]
Things to do:
* replace RFC2616 with more up-to-dates references.
* double check that we need the constants, otherwise remove them.
* attempt to figure out why respec gives me a hard time on the constants otherwise

**Comment by plehegar** [comment]
TODO+ #15

**Comment by toddreifsteck** [comment]
Looks good! Merging

**Pull request merged**